core/src/main/java/org/mule/runtime/core/internal/connection/PoolingConnectionManagementStrategy.java

```diff
@@ -218,4 +228,8 @@ private String generateId() {
     return UUID.randomUUID().toString();
   }
 
+  public boolean isJmxEnabled() {
+    return config.getJmxEnabled();
```

Comment:
is there a wat to get this from the pool itself, so the confing is not kept in memory?
MAybe the test can assesrt the mbeans instead of asserting the falg on the config?